AlexanderOtavka/ride-board
# Add user defined names

Resolves #43 

# Changes

Needs to:
 - Add "name" field to database
 - Override display name if possible with new value
 - Add "name" field to relevant user forms

# Manual Testing

> 1. *Steps for the reviewer to go through in the review app (automatically deployed on heroku and  > linked in the PR) to verify that the changes are correct.  For example:*
> 2. Go to page A and click button X
> 3. Go to page B and make sure it has feature Y

TBD
